spec/values/calculation/round/three_arguments.hrx

```diff
@@ -2,344 +2,6 @@
 Most of the same behavior tested for `calc()` applies to `round()`, but for
 terseness' sake isn't tested explicitly.
 
-<===>
```

Comment:
After adding the tests from [wpt](https://github.com/web-platform-tests/wpt/blob/master/css/css-values/round-function.html) this file was too large, so I created a strategies directory. I also added some from [this file](https://github.com/web-platform-tests/wpt/blob/master/css/css-values/round-mod-rem-computed.html#L210) to mod and rem that I thought were missing.